src/shared/components/case-viewer/services/case.resolver.ts

```diff
@@ -47,8 +49,8 @@ export class CaseResolver implements Resolve<CaseView> {
   }
 
   private navigateToCaseList() {
-    this.router.navigate(['/list/case'])
-    .then(() => this.alertService.success(CaseResolver.CASE_CREATED_MSG));
+    this.navigationNotifierService.announceNavigation({action: NavigationOrigin.NO_READ_ACCESS_REDIRECTION});
+    this.alertService.success(CaseResolver.CASE_CREATED_MSG);
```

Comment:
We are changing behaviour here no?

Reply:
right, fixing this

Reply:
I think we have to now get rid of the success above right since we moved it to the consumer.